tests/testsuite/doc.rs

```diff
@@ -297,16 +297,14 @@ fn doc_multiple_targets_same_name() {
 
     p.cargo("doc --workspace")
         .with_stderr_data(str![[r#"
-[WARNING] output filename collision.
```

Comment:
Could you adjust this so the content lines up for diffing?

Reply:
Done